content/community/getting-involved/promotion/_index.md

```diff
@@ -0,0 +1,22 @@
++++
+type = "article"
+title = "Promotion"
+date = 2021-04-14T15:16:45+10:00
+tags = ["haiku", "promotion", "getting involved", "volunteering"]
++++
+Promotion is a crucial part of ensuring Haiku continues to grow. Without spreading the word, we cannot gain new users and volunteers. 
+
+Whilst Haiku *does* have a [Promotion Team, which you can read more about here](/community/getting-involved/promotion/promotion-team/), you don't need to join the Team to promote the Project, as you will see below.
```

Comment:
This sounds like you're also actively dissuading people from joining the team. The link seems like it's a bit hard to discover to me.